torch_glow/tests/nodes/slice_test.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+from tests.utils import jitVsGlow
+
+
+def test_slice_basic():
```

Comment:
I wonder if you have the generated ir of this test? Since if operator of a (sub)graph is less than 3 ops, then nothing will be fused, fuser will just ignore it (to prevent a lot of 1-node lower graph in models)

Reply:
I didn't check the generated ir. But jitVsGlow checks whether a glowFusion node exists and whether the expected op shows up in the fusion node, so I think it should work.

Reply:
That is weird... Could you please take a look at the ir (before lower & after lower) ? you can output them in jitVsGlow by print(traced_model.graph())